client/src/components/Invitation.js

```diff
@@ -24,36 +23,37 @@ const useStyles = makeStyles((theme) => ({
     width: 400,
   },
   invitationDialogueP: {
-    fontWeight: "bold",
-    color: "#0d79de",
+    fontWeight: 'bold',
+    color: '#0d79de',
   },
   invitationDialogueTitle: {
-    textAlign: "center",
+    textAlign: 'center',
     fontSize: 22,
-    fontWeight: "bold",
-    color: "#0d79de",
+    fontWeight: 'bold',
+    color: '#0d79de',
     margin: theme.spacing(4),
   },
   invitationEmailList: {
-    marginLeft: "2em",
-    fontSize: "smaller",
+    marginLeft: '2em',
+    fontSize: 'smaller',
   },
   invitationEmailBtn: {
     fontSize: 1,
   },
 }));
 
 export default function FormDialog() {
+  const userId = '5fadeb4e66d8372cd6d05d89';
   const classes = useStyles();
-  const inputRef = React.createRef();
-  const [email, setEmail] = React.useState("");
-  const [open, setOpen] = React.useState(false);
+  const inputRef = createRef();
+
+  const [email, setEmail] = useState('');
+  const [emailList, setEmailList] = useState([]);
+  const [open, setOpen] = useState(false);
   //retrieve user object from DB and set ID
-  const [uniqueID, setID] = React.useState(
-    "https://www.EKLN-messenger.com/join/" +
-      (Math.floor(Math.random() * 90000000) + 10000000)
+  const [inviteUrl, setID] = useState(
```

Comment:
Do we need useState here? If it's not going to change, we can just do `const inviteUrl = ''`